crates/noirc_evaluator/src/ssa_refactor/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -42,8 +79,8 @@ pub(crate) struct AcirContext {
     /// addition.
     acir_ir: GeneratedAcir,
 
-    /// Maps an `AcirVar` to its known bit size.
-    variables_to_bit_sizes: HashMap<AcirVar, u32>,
+    /// Maps an `AcirVar` to its type.
+    variables_to_types: HashMap<AcirVar, AcirType>,
```

Comment:
Yeah this can be removed eventually -- I wanted to only patch the unsigned division case and patch the other operations like add_var and sub_var separately